src/client/components/ConsentsCommunicationCardABVariant.tsx

```diff
@@ -0,0 +1,142 @@
+import React, { FunctionComponent } from 'react';
+import { body, headline } from '@guardian/src-foundations/typography';
+import { css, SerializedStyles } from '@emotion/react';
+import { space, palette } from '@guardian/src-foundations';
+import { CheckboxGroup, Checkbox } from '@guardian/src-checkbox';
+import { from } from '@guardian/src-foundations/mq';
+
+interface CommunicationCardProps {
+  title: string;
+  body: string;
+  value: string;
+  checked: boolean;
+  image?: string;
+  cssOverrides?: SerializedStyles;
+}
+
+const communicationCard = css`
+  display: flex;
+  color: ${palette.text.ctaPrimary};
+  flex-direction: column;
+  width: 100%;
+  background-color: ${palette.background.ctaPrimary};
+  ${from.tablet} {
+    margin: 0;
+    padding: ${space[6]}px 0;
+  }
+`;
+
+const communicationCardHeadingImage = (image: string) => css`
+  background-image: url('${image}');
+  background-position: bottom 0px right 0px;
+  background-repeat: no-repeat;
+  background-size: 75%;
+`;
+
+const communicationCardHeadingContainer = (image?: string) => css`
+  background-color: ${palette.background.ctaPrimary};
+
+  ${image && communicationCardHeadingImage(image)}
+
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  ${from.tablet} {
+    height: auto;
+  }
+  & p {
+    ${body.medium()};
+    font-size: 18px;
+    margin-top: 6px;
+    margin-bottom: 0;
+  }
+`;
+
+const communicationCardHeadingText = css`
+  margin: 0;
+  ${headline.medium({ fontWeight: 'bold' })};
+  letter-spacing: 0.3px;
+  ${from.tablet} {
+    ${headline.small({ fontWeight: 'bold' })};
+  }
+  ${from.desktop} {
+    ${headline.medium({ fontWeight: 'bold' })};
+  }
+`;
+
+const communicationCardBodyContainer = css`
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  flex: 1 1 auto;
+  margin-top: ${space[9]}px;
+`;
+
+const communicationCardBodyText = css`
+  ${body.small()}
+  font-size: 16px;
+  margin: 0;
+  max-width: 640px;
+  border-top: 1px solid rgba(246, 246, 246, 0.4);
+  padding-top: ${space[2]}px;
+  ${from.desktop} {
+    ${body.medium()};
+  }
+`;
+
+const communicationCardCheckboxContainer = css`
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  margin-top: ${space[4]}px;
+`;
+
+// @TODO: If this variant wins, this and possible the entire style of the card component can be set using the 'brand'
+// theme in Source.
+const checkbox = css`
+  color: inherit;
+  & label div {
+    color: inherit !important;
```

Comment:
is it possible to override without `!important`?

Reply:
Possibly, but since its ABTest code, not an issue I'm going to look into unless this variant wins.